sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/ViewDescriptorFactory.scala

```diff
@@ -136,10 +86,12 @@ private[impl] object ViewDescriptorFactory extends ComponentDescriptorFactory {
         .filter(_.javaMethod.getAnnotation(classOf[Query]) != null)
       if (annotatedMethods.isEmpty)
         throw new IllegalArgumentException(
-          "No method annotated with @Query found. " +
-          "Views should have a method annotated with @Query")
+          s"No valid query method found in class ${component.getName}. " +
+          "Views should have a method annotated with @Query and annotated exposed by a REST annotation")
```

Comment:
I had a method with the Query annotation, but was getting the message "No method annotated with @Query found". Misleading. 

Also, I think we should first check for the `@Query` annotation and then for the other methods. Once we read the `Query` method, we know the view type and can then validate the other methods. 

But we can improve it in other iterations.